### PR TITLE
WFNEWS-2118 Identify panel

### PR DESCRIPTION
**Pull request by ssylver93**
image
image
image
image

### CODE REVIEW

**Comment by yzlucas** [comment]
I dont see the active weather stations appears inside the identify list. Might have missed that?

**Review by yzlucas — CHANGES_REQUESTED** [review]
Code looks good. Just please double check to add weather station into the identify list as it is an separate call. Please add a screenshot of it once it appears in the panel.

File: client/wfnews-war/src/main/angular/src/app/components/identify-panel/identify-panel.component.scss
[the remark this answers is not in the document]
Comment (ssylver93) [reply client/wfnews-war/src/main/angular/src/app/components/identify-panel/identify-panel.component.scss:1]: I imported it now, thanks

**Comment by ssylver93** [comment]
> Code looks good. Just please double check to add weather station into the identify list as it is an separate call. Please add a screenshot of it once it appears in the panel.

@yzlucas they are there, I added another screenshot to show it

**Review by yzlucas — APPROVED** [review]

**Pull request merged**